config/initializers/simple_form.rb

```diff
@@ -65,6 +65,27 @@
     b.use :error, wrap_with: { tag: 'div', class: 'usa-error-message' }
   end
 
+  config.wrappers :borderless_uswds_radio_buttons,
+                  tag: 'fieldset',
+                  wrapper_class: 'usa-fieldset margin-bottom-4',
+                  item_wrapper_tag: nil,
+                  item_label_class: 'margin-top-0 usa-radio__label width-full text-no-wrap' do |b|
+    b.use :html5
+    b.wrapper :legend, tag: 'legend', class: 'margin-bottom-2 usa-label' do |ba|
+      ba.use :label_text
+    end
+    b.use :hint, wrap_with: { tag: 'div', class: 'usa-hint margin-bottom-05' }
+    b.wrapper :grid_row, tag: :div, class: 'grid-row margin-bottom-neg-1' do |gr|
+      gr.wrapper :grid_column_radios, tag: :div, class: 'grid-col-fill' do |gc|
+        gc.wrapper :column_wrapper, tag: :div, class: 'display-inline-block minw-full' do |cr|
+          cr.use :input, class: 'usa-radio__input'
+        end
+      end
+      gr.wrapper(:grid_column_gap, tag: :div, class: 'grid-col-4 tablet:grid-col-6') {}
+    end
+    b.use :error, wrap_with: { tag: 'div', class: 'usa-error-message' }
+  end
+
```

Comment:
This is a duplicate of the styling for radio buttons above but without the `usa-radio__input--bordered` class

Reply:
At the end of a day, it's all all Ruby, so we could extract into a proc and parameterize, something like:

```ruby
radio_button_builder = proc do |b, bordered:|
   b.use :html5
   # etc
   cr.use :input, class: [
     'usa-radio__input',
     bordered && 'usa-radio__input--bordered',
   ].filter(&:present).join(' ')
end

config.wrappers :uswds_radio_buttons,
                             # etc
                            item_label_class: 'usa-radio__label width-full text-no-wrap' do |b|
  radio_button_builder.call(b, bordered: true)
end

config.wrappers : borderless_uswds_radio_buttons,
                             # etc
                            item_label_class: 'usa-radio__label width-full text-no-wrap' do |b|
  radio_button_builder.call(b, bordered: false)
end
```

Reply:
or maybe even move the proc up to a higher level....

```ruby
radio_button_builder.call(name: :uswds_radio_buttons, config: config, bordered: true)

radio_button_builder.call(name: :borderless_uswds_radio_buttons, config: config, bordered: false)
```

Reply:
I might also suggest using the opportunity to invert the modifier, so that "bordered" is the special case, not "borderless".

I was poking around SimpleForm docs and code a bit to see if there might be a better way to parameterize, but I couldn't come up with much.

Another option could be to omit the "bordered" modifier class and add it back in where it's used, via `class: 'usa-radio__input--bordered'. But that might not a particularly elegant solution either.

Reply:
I ended up just going with a .each loop -- you can see it in 209e0c6ab3b27e13cc4ead23a93945ec2604144c